src/datatype.ts

```diff
@@ -95,7 +95,7 @@ export class Datatype {
    * @method faker.datatype.datetime
    * @param options pass min OR max as number of milliseconds since 1. Jan 1970 UTC
    */
-  datetime(options): Date {
+  datetime(options?: number | { min?: number | undefined; max?: number | undefined }): Date {
```

Comment:
Optional values don't need an extra `undefined` and we also need the `precision`  cause we forward options to `faker.datatype.number`

```suggestion
  datetime(
    options?: number | { min?: number; max?: number; precision?: number }
  ): Date {
```

Reply:
thanks, updated